splice_machine/src/main/java/com/splicemachine/derby/utils/SpliceAdmin.java

```diff
@@ -734,6 +734,7 @@ public static void SYSCS_PERFORM_MAJOR_COMPACTION_ON_TABLE(String schemaName,Str
                 throw PublicAPI.wrapStandardException(Exceptions.parseException(e));
             }
             try (Partition partition = tableFactory.getTable(Long.toString(conglomID))) {
+                partition.flush();
```

Comment:
This looks unrelated, is it necessary? I think most of the times we want a MAJOR_COMPACTION we also want a flush, but we should be aware of the change.

Reply:
Major compaction for this new functionality wouldn't clean out everything without having a flush first.  Can we expect users to always flush before compaction?  Our online docs for compaction don't say that flush is required before compaction.